docs/guides/qedma-qesem.ipynb

```diff
@@ -143,143 +167,171 @@
     "</Admonition>"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "03998691",
+   "id": "5ec8053b",
    "metadata": {},
    "source": [
-    "You can use the familiar Qiskit Serverless APIs to check your Qiskit Function workload's status or return results:"
+    "### Outputs\n",
+    "Below is an example of QESEM QPU time estimation results"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "856fe992",
+   "id": "b40fb366",
    "metadata": {},
    "outputs": [],
    "source": [
-    "print(job.status())\n",
-    "result = job.result()"
+    "PrimitiveResult(\n",
+    "    [\n",
+    "        PubResult(\n",
+    "            data = DataBin(), \n",
+    "            metadata = {\"time_estimation_sec\": 3600}  \n",
+    "        )\n",
+    "    ],\n",
+    "    metadata={}\n",
+    ")"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "e2817b13",
+   "id": "625d4e77",
    "metadata": {},
    "source": [
-    "## Custom options\n",
-    "\n",
-    "Provide the `custom_options` parameter to set additional advanced features for the QESEM function:"
+    "Below is an example of QESEM execution results. Please note that it also includes the original observables (`PubResult.metadata.observables`), making the results easier to interpret."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "c2ce65e3",
+   "id": "cc58f567",
    "metadata": {},
    "outputs": [],
    "source": [
-    "# example execute QESEM job\n",
-    "job = qesem_function.run(\n",
-    "    action = \"execute\",\n",
-    "    instance = \"hub/group/project\",\n",
-    "    pubs = [\n",
-    "        (\n",
-    "            bell,\n",
-    "            [obs1,obs2,obs3]\n",
+    "PrimitiveResult(\n",
+    "    [\n",
+    "        PubResult(\n",
+    "            data = DataBin(\n",
+    "                evs = [0.98, 0.028], \n",
+    "                stds = [1.01, 0.015], \n",
+    "                shape = (2,)\n",
+    "            ),\n",
+    "            metadata = {\n",
+    "                \"observables\": [\n",
+    "                    SparsePauliOp(\n",
+    "                        [\"IIIIZ\", \"IIIZI\", \"IIZII\", \"IZIII\", \"ZIIII\"],\n",
+    "                        coeffs = [0.2 + 0.0j, 0.2 + 0.0j, 0.2 + 0.0j, 0.2 + 0.0j, 0.2 + 0.0j],\n",
+    "                    ),\n",
+    "                    SparsePauliOp(\n",
+    "                        [\"IIIZZ\", \"ZIXII\"], \n",
+    "                        coeffs=[1.0 + 0.0j, 0.5 + 0.0j]\n",
+    "                    ),\n",
+    "                ]\n",
+    "            },\n",
     "        )\n",
     "    ],\n",
-    "    precision = 0.03,\n",
-    "    run_options = {\n",
-    "        \"backend_name\": \"ibm_brisbane\",\n",
-    "    },\n",
-    "    custom_options = {\n",
-    "        \"max_qpu_time\": 14400,\n",
-    "        \"transpilation_level\": 0\n",
-    "    }\n",
-    ")"
+    "    metadata={}\n",
+    ")\n"
```

Comment:
I think it'd be helpful to have code showing how to use the primitive results, instead of showing only the type. E.g. `print(f"Expectation value from pub 0 is {result[0].data.evs}")`